index.html

```diff
@@ -79,8 +79,84 @@
           
         </div>
     </section>
-    
-    
+    <section id="about">
```

Comment:
about section on mobile lacks left and right padding. we should increase the padding on mobile view